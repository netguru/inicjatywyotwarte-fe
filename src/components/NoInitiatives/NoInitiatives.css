.no-initiatives {
  text-align: center;
  margin: 2em;
}
