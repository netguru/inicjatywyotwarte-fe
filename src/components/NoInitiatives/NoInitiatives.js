import React from 'react'
import NewButton from 'components/Buttons/NewButton/NewButton'

import './NoInitiatives.css'

export default function NoInitiatives () {
  return (
    <div className='no-initiatives'>
      <p>
        Żadna inicjatywa nie została jeszcze dodana.
        <br />
        Kliknij poniżej, aby dodać swoją
      </p>
      <NewButton />
    </div>
  )
}
