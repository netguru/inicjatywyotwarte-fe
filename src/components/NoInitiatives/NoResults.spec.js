import React from 'react'
import NoInitiatives from './NoInitiatives'
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

describe('NoInitiatives', () => {
  test('renders no initiatives div', () => {
    const wrapper = shallow(<NoInitiatives/>);
    expect(wrapper.find('div[className="no-initiatives"]').length).toEqual(1);
  });

  test('it displays no initiatives text', () => {
    const wrapper = shallow(<NoInitiatives/>);
    expect(wrapper.find('p').text()).toEqual('Żadna inicjatywa nie została jeszcze dodana.Kliknij poniżej, aby dodać swoją');
  });

  test('match snapshot', () => {
    const wrapper = shallow(<NoInitiatives/>);

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});


