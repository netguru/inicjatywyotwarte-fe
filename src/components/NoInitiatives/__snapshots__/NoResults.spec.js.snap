// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NoInitiatives match snapshot 1`] = `
<div
  className="no-initiatives"
>
  <p>
    Żadna inicjatywa nie została jeszcze dodana.
    <br />
    Kliknij poniżej, aby dodać swoją
  </p>
  <NewButton />
</div>
`;
